code/modules/uplink/uplink_items.dm

```diff
@@ -286,610 +325,505 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/stack/telecrystal/twenty
 	cost = 20
 
-// Dangerous Items
-/datum/uplink_item/dangerous
-	category = "Conspicuous and Dangerous Weapons"
+// Guns, Launchers, and other ranged weapons of destruction
+/datum/uplink_item/munitions
+	category = "Syndicate Munitions"
```

Comment:
If I recall correctly, munitions is a synonim of ammunitions, not firearms.
You'll have to rename the category and the path.

Reply:
Still, it's ambiguous and can be confused with the Ammunitions category.
`Firearms` would suffice.